packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -109,7 +122,18 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
               {originFromZero && (
                 <span className={css(classNames.zeroTick)} style={this._getStyleUsingOffsetPercent(vertical, zeroOffsetPercent)} />
               )}
-              <span ref={this._thumb} className={classNames.thumb} style={this._getStyleUsingOffsetPercent(vertical, thumbOffsetPercent)} />
+              {enableMarks && this._addTickmarks(css(classNames.regularTick))}
```

Comment:
Remove all `css` function calls that have only one argument. This helper is for cases where you need to merge 2 or more classnames.